src/v0/destinations/mp/data/MPIdentifyConfig.json

```diff
@@ -1,37 +1,54 @@
 [
   {
     "destKey": "$created",
-    "sourceKeys": "createdAtOnly",
-    "required": false,
-    "sourceFromGenericMap": true
+    "sourceKeys": ["createdAt","traits.createdAt", "context.traits.createdAt"],
```

Comment:
Do we support createdAt at root level in any RudderStack doc?

Reply:
Same for other mappings updated in this PR.

Reply:
No we do not support createdAt at the root level. This is used for 
`let sentOnceTransform = constructPayload(setOnceEligible, mPIdentifyConfigJson);` in `trimTraits` function. In this function `setOnceEligible` object , which we are internally making, will not have traits or context.traits object. So we are finding it from the root level of  `setOnceEligible`

Reply:
kept the root level properties priority as least. Do you still see there can be issues. In that case will make a different mappingJSON file. will ask inputs from @utsabc / @krishna2020  here as well

Reply:
based on discussion with @utsabc added a dedicated mappingJSON for setOnce properties which we will be tracked internally.